ArrayIndexOutOfBoundsException in histogram call for uint16 layer

Let’s figure out if this is an error in the dataset, in wkw or in the webknossos backend

```
backend: ! @7efb7ah3e - Internal server error, for (GET) [/data/datasets/sample_organization/data_types/layers/color_uint16/histogram?token=rxBkERKXmWLU6iMAwNrNwg] ->
backend:  
backend: play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[ArrayIndexOutOfBoundsException: null]]
backend: 	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:351)
backend: 	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:267)
backend: 	at play.core.server.AkkaHttpServer$$anonfun$1.applyOrElse(AkkaHttpServer.scala:448)
backend: 	at play.core.server.AkkaHttpServer$$anonfun$1.applyOrElse(AkkaHttpServer.scala:446)
backend: 	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:413)
backend: 	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
backend: 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
backend: 	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
backend: 	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:92)
backend: 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
backend: Caused by: java.lang.ArrayIndexOutOfBoundsException: null
backend: 	at com.scalableminds.webknossos.datastore.services.BinaryDataService.$anonfun$cutOutCuboid$4(BinaryDataService.scala:135)
backend: 	at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:155)
backend: 	at com.scalableminds.webknossos.datastore.services.BinaryDataService.$anonfun$cutOutCuboid$3(BinaryDataService.scala:121)
backend: 	at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:155)
backend: 	at com.scalableminds.webknossos.datastore.services.BinaryDataService.$anonfun$cutOutCuboid$2(BinaryDataService.scala:119)
backend: 	at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:155)
backend: 	at com.scalableminds.webknossos.datastore.services.BinaryDataService.$anonfun$cutOutCuboid$1(BinaryDataService.scala:118)
backend: 	at com.scalableminds.webknossos.datastore.services.BinaryDataService.$anonfun$cutOutCuboid$1$adapted(BinaryDataService.scala:97)
backend: 	at scala.collection.immutable.List.foreach(List.scala:388)
backend: 	at com.scalableminds.webknossos.datastore.services.BinaryDataService.cutOutCuboid(BinaryDataService.scala:97) 
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -19,6 +19,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 
 ### Fixed
 - Fixed opening view only dataset links with arbitrary modes being initially displayed in plane mode. [#4421](https://github.com/scalableminds/webknossos/pull/4421)
+- Fixed the creation of histograms for float datasets that only have one value besides 0. [#4468](https://github.com/scalableminds/webknossos/pull/4468)
 
 ### Removed
 -

--- a/webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/FindDataService.scala
+++ b/webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/FindDataService.scala
@@ -263,7 +263,8 @@ class FindDataService @Inject()(dataServicesHolder: BinaryDataServiceHolder)(imp
               case ((currMin, currMax), e) => (math.min(currMin, e), math.max(currMax, e))
             }
             val bucketSize = (max - min) / 255
-            floatData.foreach(el => counts(Math.roundDown((el - min) / bucketSize)) += 1)
+            val finalBucketSize = if (bucketSize == 0f) 1f else bucketSize
+            floatData.foreach(el => counts(Math.roundDown((el - min) / finalBucketSize)) += 1)
             extrema = (min, max)
         }
       }

